apis/ui-apis.yaml

```diff
@@ -0,0 +1,67 @@
+kind: API
+name: get-aws
+enabled: true
+function: aws
+cors: true
+methods:
+  - GET
+protocols:
+  - https
+hosts:
+  - ato-api.dispatchframework.io
```

Comment:
We could just remove the hosts.  The API will use the org as a path based namespace

Reply:
went with a generic hostname